src/calculateRentalCost.js

```diff
@@ -1,10 +1,12 @@
-/**
- * @param {number} days
- *
- * @return {number}
- */
 function calculateRentalCost(days) {
-  // write code here
-}
+  let totalCost = days * 40;
```

Comment:
Use `const` for `totalCost` if you're not reassigning it. In this case, you are reassigning it, but consider using a different approach where you can use `const` instead.